packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -88,10 +88,10 @@ class SpeedDialAction extends React.Component {
         onTouchStart: () => {
           startTime = new Date();
         },
-        onTouchEnd: () => {
+        onTouchEnd: e => {
```

Comment:
I believe use `event` in preference to `e` elsewhere in the codebase.

Reply:
Changed from `e` to `event` to better align with codebase